### PR TITLE
finish project

### PR DESCRIPTION
**Pull request by supert111**
### popup-html-css

[Demo](https://supert111.github.io/popup-html-css/) |
[Code base](https://github.com/supert111/popup-html-css)

### CODE REVIEW

**Comment by supert111** [comment]
Hello, initially I did not see in the documentation what structure. There are no errors in validation.

**Comment by supert111** [comment]
Please Re-request review

File: submissions/supert111/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        <div class="container">
+            <nav class="navBlock">
+                <ul class="list">
+                    <li class="link"><a class="linkFokus" href="">Mail</a></li>
+                    <li class="link"><a class="linkFokus" href="">Images</a></li>
+                </ul>
+                <input class="checkInput" type="checkbox" id="checkInputId">
+                <label class="check" for="checkInputId">
+                    <span class="checkBox"></span>
+                </label>
```
Comment (A-Ostrovnyy) [inline submissions/supert111/html-css-popup/index.html:21]: The menu icon is a part of navigation too, put it into list.

File: submissions/supert111/html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+                <div class="wrapperPopup">
+                    <div class="popup">
+                        <ul class="popupList">
+                            <li class="popupItem">
+                                <a class="linkFokusPopupItem" href="">
+                                    <img class="popupImg" src="images/calendar.png" alt="calendar" width="40">
```
Comment (A-Ostrovnyy) [inline submissions/supert111/html-css-popup/index.html:27]: Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!

File: submissions/supert111/html-css-popup/index.html [hunk 3]
```diff
@@ -14,134 +14,136 @@
...
+                                        </li>
+                                        <li class="popupItem noCheckItem">
+                                            <a class="linkFokusPopupItem" href=""> 
+                                                <img class="popupImg" src="images/mars.png" alt="icon-mars" width="40">
+                                                <p>Mars</p>
+                                            </a>
+                                        </li>
+                                        <li class="popupItem noCheckItem">
+                                            <a class="linkFokusPopupItem" href=""> 
+                                                <img class="popupImg" src="images/collection.png" alt="icon-collection" width="40">
```
Comment (A-Ostrovnyy) [inline submissions/supert111/html-css-popup/index.html:122]: You have text inside your links + icons are decorative images so `alt` attribute should be empty

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
One last issue.

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**